src/singletons/WindowManager.cpp

```diff
@@ -297,18 +297,29 @@ void WindowManager::initialize(Settings &settings, Paths &paths)
     assert(!this->initialized_);
 
     {
-        auto windowLayout = this->loadWindowLayoutFromFile();
+        WindowLayout windowLayout;
+
+        if (getArgs().customChannelLayout)
+        {
+            // We would love to use std::optional<T>::value() instead but macOS
+            // does not support it yet. :)
```

Comment:
But it does though :) (from macOS 10.14 (apple clang 10.0)). Right now chatterino min macOS version is 10.13 for some reason, we could just bump it to 10.14 or even 10.15 :)

Reply:
Although I agree an argument could be made to explicitly mention in the comment that this refers to 10.13

Reply:
It's not actually about macos version, it's about apple clang version. So you could just use normal compiler and you will be fine :) Also, i think a good point to raise version is that some of the external libs (for example boost) already build for 10.15 and it always gives a warning that it's linked to a different version. Sorry for write it here, kind of offtopic

Reply:
> It's not actually about macos version, it's about apple clang version.

Aha, I am little confused about that since the error message explicitly mentions 10.14:
```
../src/singletons/WindowManager.cpp:304:58: error: 'value' is unavailable: introduced in macOS 10.14
```

But then again, I am just vaguely familiar with the Apple ecosystem so I don't feel like I can contribute much to the conversation

Reply:
Ye, i agree, message is definitely confusing 😄 Sorry, i just like `::value` usage here, so i thought you could just bump macOS min version :)

Reply:
I would prefer to use `::value()` as well but it's not my decision to make :man_shrugging: Would love to hear input from @fourtf and @pajlada on this

Reply:
Decided to go with `boost::optional` for now, as suggested by @pajlada on stream.